po/fr.po

```diff
@@ -50,16 +50,16 @@ msgstr "Terminer le processus"
 #: src/Widgets/Headerbar.vala:26
 #, fuzzy
```

Comment:
```suggestion
```

If you don't remove the `fuzzy` marks, your translation won't be applied to the UI; it means "this translation string is for the old source string and needs editing".

Reply:
Thanks, didn't know that !

Reply:
You don't seem to remove this line.

Reply:
Well, I'm not truly satisfied by the french translation. This is why I didn't remove the fuzzy comment.